tanzu-application-platform/README.md

```diff
@@ -0,0 +1,184 @@
+```
+
+### Let's Encrypt CA Issuer
+
+Animal Rescue enables TLS connections for security and reliable transport and utilizes the Let's Encrypt production CA. Install the Let's Encrypt
+CA issuer into your cluster by running the following command and replacing the following place holder:
+
+- **<email>** – The email address of the subject responsible for the issued certificates.
+ 
+```
+ytt -f caIssuer.yml -v email=<email> | kubectl apply -f-
+```
+
+For example:
+
+```
+ytt -f caIssuer.yml -v email=joe@joesgarage.com | kubectl apply -f-
+```
+
+### AppSSO Deployment
+
+Animal Rescue requires the use of an AppSSO authorization server and client registration resource. 
+
+Deploy the authorization server instance by running the following commands and replacing these place holders:
+
+- **<serviceNamespace>** – Namespace where service instances will be deployed
+- **<devDefaultAccountUsername>** – Username for authentication
+- **<devDefaultAccountPassword>** – Password for authentication
+
+```
+ytt -f appSSOInstance.yml -v serviceNamespace=<serviceNamespace> -v devDefaultAccountUsername=<devDefaultAccountUsername> -v devDefaultAccountPassword=<devDefaultAccountPassword> | kubectl apply -f-
+```
+
+For example:
+
+```
+ytt -f appSSOInstance.yml -v serviceNamespace=service-instances -v devDefaultAccountUsername=animal -v devDefaultAccountPassword=rescue | kubectl apply -f-
+```
+
+Next, create a ClassClaim resource by running the following command and replacing the `<workloadNamespace>` placeholder with the namespace where the application will be deployed. 
+This will also created the secret that will contain the AppSSO credential information that will used by the Spring Cloud Gateway.
+
+- **<workloadNamespace>** – Namespace where the application will be deployed
+
+```
+ytt -f workloadRegistrationResource.yml -v workloadNamespace=<workloadNamespace>| kubectl apply -f-
+```
+
+For example:
+
+```
+ytt -f workloadRegistrationResource.yml -v workloadNamespace=workloads | kubectl apply -f-
+```
+
+Next, obtain the appSSO issuerURI by running the following command replacing <serviceNamespace> with the name of the namespace where the service instance 
+was deployed:
+
+```
+kubectl get authserver -n <serviceNamespace>
+```
+
+Save the Issuer URI as you may need to create a new entry in your DNS registrar for this URL; this will likely be
+necessary if you are NOT using wildcard records in your DNS registrar.  DNS is also required in order for the TLS certificate to be issued for the auth server.
+
+Finally, obtain the appSSO credential secret name by running the following command replacing` <workloadNamespace>` placeholder with the namespace where the application will be deployed.  Save the `status.claimed-resource.name` field value as this will be the secret name that will need to provided when configuring the Spring Cloud Gateway.
+
+```
+tanzu service class-claim get appsso-animal-rescue -n <workloadNamespace>|
```

Comment:
I think this might have a trailing "|", or if you were planning on adding in the command to just extract the Name. Which I think would be the below 

tanzu service class-claim get appsso-animal-rescue -n <workloadNamespace> | yq '.Status.["Claimed Resource"].Name'

Reply:
The trailing "|" should not be there.  Resolved now in the PR.